docs/src/components/base.css

```diff
@@ -0,0 +1,226 @@
+html {
```

Comment:
Out of interest where is the base/core CSS from?

Reply:
gatsby included "layout.css". it had a bunch of stuff that was ugly so I stripped it back to create this monster. we need to replace the whole thing with a design system